internal/infrastructure/storage/db/pg/migration/20230504102232_init.up.sql

```diff
@@ -0,0 +1,64 @@
+CREATE TABLE market (
+    name VARCHAR(50) NOT NULL PRIMARY KEY,
+    base_asset VARCHAR(64) NOT NULL,
+    quote_asset VARCHAR(64) NOT NULL,
+    base_asset_precision INTEGER,
+    quote_asset_precision INTEGER,
+    tradable BOOLEAN,
+    strategy_type INTEGER,
+    base_price DOUBLE PRECISION,
+    quote_price DOUBLE PRECISION,
+    UNIQUE (base_asset, quote_asset)
+);
+
+CREATE TABLE fee (
+    id SERIAL PRIMARY KEY,
+    base_asset_fee BIGINT NOT NULL,
+    quote_asset_fee BIGINT NOT NULL,
+    type VARCHAR(10) NOT NULL CHECK (type IN ('percentage', 'fixed')),
+    fk_market_name VARCHAR(50) NOT NULL,
+    FOREIGN KEY (fk_market_name) REFERENCES market (name)
+);
+
+CREATE TABLE trade (
+    id VARCHAR(50) PRIMARY KEY,
+    type INTEGER NOT NULL,
+    fee_asset VARCHAR(255) NOT NULL,
+    fee_amount BIGINT NOT NULL,
+    trader_pubkey BYTEA,
+    status_code INTEGER,
+    status_failed BOOLEAN,
```

Comment:
these fields don't need defaults but they must be not null.
```suggestion
    status_code INTEGER NOT NULL,
    status_failed BOOLEAN NOT NULL,
```